app/src/main/java/com/futurumgame/base/interfaces/INormedNotation.java

```diff
@@ -0,0 +1,12 @@
+package com.futurumgame.base.interfaces;
+
+public interface INormedNotation {
+
+    String NaN = "NaN";
```

Comment:
Would be better to make those thingies constants.

Reply:
What do you mean by this

Reply:
Make those variables final since they aren't supposed to be changed...